src/utils/propUtils.js

```diff
@@ -60,3 +63,15 @@ export const customPropTypes = {
   },
 }
 
+// ------------------------------------
+// Prop Renderers
+// ------------------------------------
+export const iconPropRenderer = (val) => _.isString(val) ? <Icon className={val} /> : val
+export const imagePropRenderer = (val) => _.isString(val) ? <Image src={val} /> : val
+
+// ------------------------------------
+// Prop To Class Name
+// ------------------------------------
+export const useKeyOnly = (val, key) => val && key
+export const useValueAndKey = (val, key) => val && val !== true && `${val} ${key}`
+export const useKeyOrValueAndKey = (val, key) => val && (val === true ? key : `${val} ${key}`)
```

Comment:
This information is prime. Perhaps it might be a good idea to document it somewhere in the lib like I mentioned above.